commands/root.go

```diff
@@ -42,3 +45,39 @@ func (c *rootCmd) persistentPreRun(cmd *cobra.Command, args []string) {
 		pretty.DisableSpinner()
 	}
 }
+
+func (c *rootCmd) runE(cmd *cobra.Command, args []string) error {
+	cfg, err := config.Global()
+	if err != nil {
+		return err
+	}
+
+	db, err := database.NewServiceDB(cfg.Database.Path)
+	if err != nil {
+		return err
+	}
+
+	logger.Init(cfg.Log.Format, cfg.Log.Level)
+
+	logrus.Println("Starting MESG Core", version.Version)
+
+	for _, plugin := range cfg.Server.Plugins {
+		if plugin.Path != "" {
+			if _, err := service.NewFromPath(plugin.Path); err != nil {
+				return err
+			}
+		}
+	}
+
+	tcpServer := &grpc.Server{
```

Comment:
initializing of core should be made inside package core. _core package_ can have a func named _New_. It can have _Start_ and _Close_ methods. This way we can simply create a binary from a main program that uses _core package_ to start core and we can distribute that binary to world. or/and we can use _core package_ from _mesg-core_ cli to start core with a command.